lib/validate.js

```diff
@@ -12,25 +12,25 @@ function validate (state, doc) {
   .then(function () {
     return state.validate(doc)
   })
-  .catch(function (error) {
-    var err = new Error()
+  .catch(function (rejectValue) {
+    var error = new Error()
 
-    if (!(error instanceof Error)) {
-      if (typeof error === 'string') {
-        err.message = error
+    if (!(rejectValue instanceof Error)) {
+      if (typeof rejectValue === 'string') {
+        error.message = rejectValue
       } else {
-        err.message = 'check error value for more details'
-        err.value = error
+        error.message = 'check error value for more details'
+        error.value = rejectValue
       }
     } else {
-      if (!error.message) {
-        err.message = 'your doc(s) failed validation'
+      if (!rejectValue.message) {
+        error.message = 'your doc(s) failed validation'
```

Comment:
I would phrase the default error message without using a pronoun. Maybe something like `document validation failed`

Reply:
@mbad0la can you make that change?

Reply:
Sure, I'm just waiting on us to resolve how to go about Error creation/mapping. This is pretty straight-forward to fix.